logoapp/js/line_data.js

```diff
@@ -473,20 +473,65 @@ LineAppFramework.prototype.onClickUploadButton = function() {
 	encoder.finish();
 	var bin = new Uint8Array(encoder.stream().bin);
 	var blob = new Blob([bin.buffer], {type:'image/gif'});
+	var b64 = window.btoa(encoder.stream().getData());
 
 	//download blob
-	var FileName = "lineanime.gif"
+	//var FileName = "lineanime.gif"
+
+	clearInterval(this.completeLoopInterval	);
+	var uploadData = new Array(8);
+	for(var i=0; i<uploadData.length; i++) {

```

Comment:
__[jshint]__
*  'i' is already defined.


<!-- Comment by [SideCI](https://sideci.com) -->